src/helpers/users.js

```diff
@@ -55,7 +55,23 @@ exports.formatQueryForUsersList = async (query) => {
   }
 
   if (query.company) {
-    const users = await UserCompany.find({ company: query.company }, { user: 1 }).lean();
+    const users = await UserCompany
```

Comment:
cf mon retour sur les tests, ici il faut que : 
- avant ton appel `UserCompany.find` tu définis `const userIds = []`
- ici tu appelles ta constante `usercompanies`
- après l'appel, tu fais `userIds.push(...usercompanies.map(uc => uc.user)`
- dans le `if (query.includesHoldingAdmins)`, tu fais `userIds.push(...userHoldings.map(uh => uh.user)`
- et enfin, `formattedQuery._id = { $in: userIds };

Reply:
si tu fais ca avant de changer ton test, il devrait echoué du coup

Reply:
done